Deploy `cert-manager` on Google Kubernetes Engine Tutorial - remove `google domains` (#2062)
cert-manager/website: content/docs/tutorials/getting-started-with-cert-manager-on-google-kubernetes-engine-using-lets-encrypt-for-ingress-ssl/README.md

description: Learn how to deploy cert-manager on Google Kubernetes (GKE) Engine and then configure it to sign SSL certificates using Let's Encrypt
---

> *Last Verified: 29 July 2026*

Reviewer: Bumping this date tells readers the whole tutorial was re-verified end-to-end on GKE, but only these two sections were reworded — and the clean up step is currently broken (see review body). Suggest keeping the old date until someone re-runs the tutorial:

```suggestion
*Last Verified: 6 June 2025*
```

**💻 Domain Name**

> You will need a domain name and the ability to create DNS records in that domain. NameCheap and GoDaddy are two well-known registrars that may be used to register your domain name.

Author: do we want to call out any registrars here or just say something like "you can use any registrar"

Reviewer: I'd keep it generic — naming registrars dates the doc the same way the Google Domains walkthrough did. Something like: "You will need a domain name and the ability to create DNS records in that domain, using any domain name registrar." If you do keep the examples, note the brand spells itself "Namecheap", not "NameCheap".

*with claude fable-5*

## 4. Create a domain name for your website

> You will need a domain name for your website, this allows Let's Encrypt to check your domain before it signs your SSL certificate,

Reviewer: As Copilot noted, this is a comma splice. It also slightly misattributes the mechanism: Let's Encrypt checks that the domain is reachable, the name itself doesn't "allow" anything. The original wording on master was fine and flows into the next line:

```suggestion
You will need a domain name for your website and Let's Encrypt checks your domain before it signs your SSL certificate,
```

so the domain name needs to be reachable from the Internet.

Once you have purchased your desired domain name, save it in an environment variable:

```bash

> export DOMAIN_NAME=mydomain.net

Reviewer: `mydomain.net` is a real, registrable domain — not reserved like `example.net` ([RFC 2606](https://datatracker.ietf.org/doc/html/rfc2606)) — so a copy-pasting reader points the rest of the tutorial (curl, Ingress host, ACME order) at somebody else's domain. Matching the pattern used for `IP_ADDRESS` above:

```suggestion
export DOMAIN_NAME=example.net  # Replace with your domain name
```

```

> Next, you will need to create a new `A` record pointing at the IP address that we created above. Head back to your domain registrar and create a add a new record of type `A` that points to the IP address from the previous step.

Reviewer: Three things in this sentence:

1. The typo Copilot flagged: "create a add a".
2. "Head back" is a leftover from the deleted Google Domains flow — nothing sends the reader to the registrar any more.
3. Most important: the removed text said to leave "Host name" **empty** because we are configuring the top-level domain. Without that instruction, a reader who accepts their registrar's default host (often `www`) creates `www.mydomain.net`, and then `curl http://$DOMAIN_NAME`, the Ingress, and the ACME challenge for `$DOMAIN_NAME` all fail with NXDOMAIN.

One suggestion covering all three:

```suggestion
Next, you will need to create a new `A` record pointing at the IP address that we created above. Go to your domain registrar's DNS settings and add a new record of type `A` that points to the IP address from the previous step. Leave the host name empty (some registrars use `@` for this) because we are configuring the top-level domain name.
```
